packages/frontend-v2/utils/plan/preparePlanForDnd.ts

```diff
@@ -0,0 +1,97 @@
+import {
```

Comment:
functions moved from useStudentWithPlan, can ignore if you https://github.com/sandboxnu/graduatenu/pull/439 was reviewed.